### PR TITLE
second commit

### PR DESCRIPTION
**Pull request by Sindhujarajen**

### CODE REVIEW

File: first_project/src/app/header/header.component.html [hunk 1]
```diff
@@ -34,8 +34,20 @@ <h1 appHighlight>Muraai Information Technology</h1>
...
-                (click)="snav.toggle()">goshopping</a>
+        
+                <mat-accordion appHighlight background="pink">
+                    <mat-expansion-panel >
+                      <mat-expansion-panel-header>
+                        <mat-panel-title>
+                            <a mat-list-item appHighlight background="pink" routerLink="/goshopping"(click)="snav.toggle()">goshopping  </a>
+                        </mat-panel-title>
+                      </mat-expansion-panel-header>
+                      <p> <a mat-list-item appHighlight background="orange" routerLink="/goshopping"(click)="snav.toggle()">Clothes</a></p>
```
Comment (Sushmitha-Vk) [inline first_project/src/app/header/header.component.html]: there is no child routes still...?

**Review by Sushmitha-Vk — CHANGES_REQUESTED** [review]

File: first_project/src/app/header/header.component.html (hunk 1 above)
Comment (Sindhujarajen) [reply first_project/src/app/header/header.component.html]: resolved mam

File: first_project/src/app/goshopping/goshopping.component.ts [hunk 2]
```diff
@@ -1,23 +1,55 @@
...
+  constructor(private service: SampleserviceService,private route:Router) {
+   }
+  ngOnDestroy(): void {
+    this.onDestroy$.next(true);
+    this.onDestroy$.complete();
+  }
   ngOnInit() {
-    this.service.getShopping().subscribe((code: any) => {
-      this.cards = code
+    // this.service.getClothes().pipe(takeUntil(this.onDestroy$)).subscribe((code: any) => {
```
Comment (Sushmitha-Vk) [inline first_project/src/app/goshopping/goshopping.component.ts]: remove commented codes

File: first_project/src/app/goshopping/goshopping.component.ts [hunk 3]
```diff
@@ -1,23 +1,55 @@
...
+    //   this.cards = code
       // console.log(this.cards);
-    })
+      this.route.events.pipe(filter(event=>event instanceof NavigationStart)).subscribe((routelink:any) => {
+       console.log('-----',routelink)
```
Comment (Sushmitha-Vk) [inline first_project/src/app/goshopping/goshopping.component.ts]: remove console.logs

**Review by Sushmitha-Vk — CHANGES_REQUESTED** [review]

File: first_project/src/app/goshopping/goshopping.component.ts (hunk 2 above)
Comment (Sindhujarajen) [reply first_project/src/app/goshopping/goshopping.component.ts]: resolved mam

File: first_project/src/app/goshopping/goshopping.component.ts (hunk 3 above)
Comment (Sindhujarajen) [reply first_project/src/app/goshopping/goshopping.component.ts]: resolved mam

**Review by Sushmitha-Vk — APPROVED** [review]

**Comment by Sushmitha-Vk** [comment]
resolve conflicts

**Review by Sushmitha-Vk — APPROVED** [review]

**Pull request merged**